emu/memory.h

```diff
@@ -113,6 +114,7 @@ int pt_copy_on_write(struct mem *src, struct mem *dst, page_t start, page_t page
 #define MEM_WRITE 1
 // Must call with mem read-locked.
 void *mem_ptr(struct mem *mem, addr_t addr, int type);
```

Comment:
How about a new value for `type` (maybe `MEM_WRITE_PTRACE`) instead of a new boolean argument?